### PR TITLE
lang: Improve Polish localization.

### PR DESCRIPTION
**Pull request by mavethee**
Fully updated Polish localization basing on English xml.
- Added and translated missing strings
- Corrected eventual mistakes

### CODE REVIEW

File: lang/system/pl.xml [hunk 1]
```diff
@@ -55,9 +114,12 @@ Kod błędu: {}</an_error_occurred>
 		<file_corrupted>Plik jest uszkodzony.</file_corrupted>
 		<microphone_disabled>Włącz mikrofon.</microphone_disabled>
 		<no>Nie</no>
+		<ok>OK</ok>
```
Comment (Croden1999) [inline lang/system/pl.xml:117]: no need add it, is same exist in English.

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/system/pl.xml [hunk 2]
```diff
@@ -40,11 +88,22 @@
 			<level>Poziom</level>
 			<name>Nazwa</name>
 			<trophy_earning>Zdobywanie trofeów</trophy_earning>
-			<parental_controls>kontrola rodzicielska</parental_controls>
+			<parental_controls>Kontrola rodzicielska</parental_controls>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:91]: Can you check it ? @Croden1999

File: lang/system/pl.xml [hunk 3]
```diff
@@ -188,32 +250,97 @@ Kod błędu: {}</an_error_occurred>
 		</states>
 	</compatibility>
 
-	<compat_db>
+	<compat_db name="Baza kompatybilności"/>
```
Comment (Croden1999) [inline lang/system/pl.xml:253]: revert it, compat_db not used the "name".

File: lang/system/pl.xml [hunk 4]
```diff
@@ -164,22 +226,22 @@ Kod błędu: {}</an_error_occurred>
...
 			<days>31</days>
 		</small_mday>
-		<hidden_trophy>Skjult trophy</hidden_trophy>
-		<one_hour_ago>1 godz. temu</one_hour_ago>
-		<one_minute_ago>1 min temu</one_minute_ago>
-		<bronze>Bronse</bronze>
-		<gold>Gull</gold>
-		<platinum>Platina</platinum>
-		<silver>Sølv</silver>
+		<hidden_trophy>Ukryte trofeum</hidden_trophy>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:229]: No change this, is fine already

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/system/pl.xml [hunk 5]
```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
 				<mm_dd_yyyy>MM/DD/RRRR</mm_dd_yyyy>
 			</date_format>
 			<time_format name="Format godziny">
-				<clock_12_hour>Zegar 12-godz</clock_12_hour>
-				<clock_24_hour>Zegar 24-godz</clock_24_hour>
+				<clock_12_hour>Zegar 12-godzinowy</clock_12_hour>
+				<clock_24_hour>Zegar 24-godzinowy</clock_24_hour>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:560]: Don't touch it, revert change for already translate already present

File: lang/system/pl.xml [hunk 6]
```diff
@@ -275,30 +422,131 @@ Nie wyłączaj systemu ani nie zamykaj aplikacji.</warning_saving>
...
+			<hide_show_control>Lewy przycisk myszy lub wciśnij Trójkąt</hide_show_control>
+			<exit_manual>Wyjdź z instrukcji</exit_manual>
+			<exit_manual_control>Naciśnij Esc lub wciśnij PS na kontrolerze</exit_manual_control>
+		</help>
 	</live_area>
+
+	<performance_overlay>
+		<avg>Średnia (Avg)</avg>
+		<min>Mimimum (Min)</min>
+		<max>Maksimum (Max)</max>
```
Comment (Croden1999) [inline lang/system/pl.xml:525]: The text exceeds the dialog

File: lang/system/pl.xml (hunk 2 above)
Comment (Croden1999) [reply lang/system/pl.xml:91]: He is right.

File: lang/system/pl.xml [hunk 7]
```diff
@@ -357,25 +829,30 @@ System Vita3K jest gotowy!</completed_setup>
 		<content_manager>Menedźer zawartości</content_manager>
 	</sys_apps_title>
 
-	<trophy_collection>
+	<trophy_collection name="Kolekcja trofeów">
```
Comment (Croden1999) [inline lang/system/pl.xml:832]: revert it, this line not used the "name".

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/system/pl.xml (hunk 2 above)
Comment (mavethee) [reply lang/system/pl.xml:91]: @Zangetsu38 All I did is match to rest strings displayed in this section aka start capitalized.

File: lang/system/pl.xml (hunk 4 above)
Comment (mavethee) [reply lang/system/pl.xml:229]: @Zangetsu38 In what language it was fine? XD

File: lang/system/pl.xml (hunk 5 above)
Comment (mavethee) [reply lang/system/pl.xml:560]: @Zangetsu38 if you want to use short form, use the damn dot.
Polish langauge has rule, if shortcut does not end with last letter of this word, needs a dot at the end.

File: lang/system/pl.xml (hunk 1 above)
Comment (mavethee) [reply lang/system/pl.xml:117]: As it's updated God's know when, it was easier to follow and compare with English file, that will be always up to date.

File: lang/system/pl.xml (hunk 4 above)
Comment (nishinji) [reply lang/system/pl.xml:229]: His change is right

File: lang/system/pl.xml [hunk 8]
```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
...
-						<lang id="1024">Rosyjski</lang>
-						<lang id="2048">Fiński</lang>
-						<lang id="4096">Szwedzki</lang>
-						<lang id="524288">Turkish</lang>
+						<lang id="131072">Portugalska (Brazylia)</lang>
+						<lang id="512">Portugalska (Portugalia)</lang>
+						<lang id="1024">Rosyjska</lang>
+						<lang id="2048">Fińska</lang>
+						<lang id="4096">Szwedzka</lang>
+						<lang id="524288">Turecka</lang>
```
Comment (nishinji) [inline lang/system/pl.xml:583]: Here is baed on vita, so you need to chack vita (slightly wrong)

File: lang/system/pl.xml (hunk 5 above)
Comment (Zangetsu38) [reply lang/system/pl.xml:560]: Is official translation, juste revert

File: lang/system/pl.xml (hunk 4 above)
Comment (Zangetsu38) [reply lang/system/pl.xml:229]: Translate is from psvita directy

Comment (nishinji) [reply lang/system/pl.xml:229]: I mean, croden and my typo
so keep this change

File: lang/system/pl.xml [hunk 9]
```diff
@@ -164,22 +226,22 @@ Kod błędu: {}</an_error_occurred>
...
-		<gold>Gull</gold>
-		<platinum>Platina</platinum>
-		<silver>Sølv</silver>
+		<hidden_trophy>Ukryte trofeum</hidden_trophy>
+		<one_hour_ago>godzinę temu</one_hour_ago>
+		<one_minute_ago>minutę temu</one_minute_ago>
+		<bronze>Brązowe</bronze>
+		<gold>Złote</gold>
+		<platinum>Platynowe</platinum>
+		<silver>Srebrne</silver>
```
Comment (nishinji) [inline lang/system/pl.xml:235]:
```suggestion
		<bronze>Brązowy</bronze>
		<gold>Złoty</gold>
		<platinum>Platynowy</platinum>
		<silver>Srebrny</silver>
```

File: lang/system/pl.xml (hunk 2 above)
Comment (Zangetsu38) [reply lang/system/pl.xml:91]: ok, it is fine, no idea why have error here

File: lang/system/pl.xml (hunk 9 above)
Comment (Zangetsu38) [reply lang/system/pl.xml:235]: yes, this is correct 👌

File: lang/system/pl.xml [hunk 10]
```diff
@@ -164,22 +226,22 @@ Kod błędu: {}</an_error_occurred>
...
 		<hours_ago>{} godz. temu</hours_ago>
-		<minutes_ago>{} min temu</minutes_ago>
+		<minutes_ago>{} min. temu</minutes_ago>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:237]: for it, revert, is already fine, no have . for min word

File: lang/system/pl.xml (hunk 6 above)
Comment (mavethee) [reply lang/system/pl.xml]: Added polish shortcuts then, they are much the same just avg is different

File: lang/system/pl.xml (hunk 10 above)
Comment (mavethee) [reply lang/system/pl.xml]: Corrected.

File: lang/system/pl.xml (hunk 7 above)
Comment (mavethee) [reply lang/system/pl.xml]: Done, my bad.

File: lang/system/pl.xml [hunk 11]
```diff
@@ -164,22 +226,22 @@ Kod błędu: {}</an_error_occurred>
...
-		<hidden_trophy>Skjult trophy</hidden_trophy>
-		<one_hour_ago>1 godz. temu</one_hour_ago>
-		<one_minute_ago>1 min temu</one_minute_ago>
-		<bronze>Bronse</bronze>
-		<gold>Gull</gold>
-		<platinum>Platina</platinum>
-		<silver>Sølv</silver>
+		<hidden_trophy>Ukryte trofeum</hidden_trophy>
+		<one_hour_ago>godzinę temu</one_hour_ago>
+		<one_minute_ago>minutę temu</one_minute_ago>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:231]: You have no understund, just no hange it, is already fine

File: lang/system/pl.xml [hunk 12]
```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
 				<mm_dd_yyyy>MM/DD/RRRR</mm_dd_yyyy>
 			</date_format>
 			<time_format name="Format godziny">
-				<clock_12_hour>Zegar 12-godz</clock_12_hour>
-				<clock_24_hour>Zegar 24-godz</clock_24_hour>
+				<clock_12_hour>Zegar 12-godz.</clock_12_hour>
+				<clock_24_hour>Zegar 24-godz.</clock_24_hour>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:560]: Revert both here, is totaly fine, no using . In end

File: lang/system/pl.xml [hunk 13]
```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
...
 			</time_format>
 		</date_time>
 		<language name="Język">
 			<system_language>Język systemu</system_language>
-			<input_language name="język wprowadzania">
-				<keyboards name="Klawiatury">
+			<input_language name="Języki wprowadzania">
+				<keyboards name="Rodzaj klawiatury">
```
Comment (Zangetsu38) [inline lang/system/pl.xml:566]: Revert it, is good, no have two word

File: lang/system/pl.xml [hunk 14]
```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
...
-						<lang id="1024">Rosyjski</lang>
-						<lang id="2048">Fiński</lang>
-						<lang id="4096">Szwedzki</lang>
-						<lang id="524288">Turkish</lang>
+						<lang id="131072">Portugalska (Brazylia)</lang>
+						<lang id="512">Portugalska (Portugalia)</lang>
+						<lang id="1024">Rosyjska</lang>
+						<lang id="2048">Fińska</lang>
+						<lang id="4096">Szwedzka</lang>
+						<lang id="524288">Turecka</lang>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:583]:
```suggestion
						<lang id="524288">Turecki</lang>
```

Comment (Zangetsu38) [reply lang/system/pl.xml:583]: Revert all other, is already good

File: lang/system/pl.xml [hunk 15]
```diff
@@ -358,24 +830,29 @@ System Vita3K jest gotowy!</completed_setup>
...
+		<locked>Niezdobyto</locked>
 		<details>Szczegóły</details>
 		<earned>Zdobyto</earned>
 		<name>Nazwa</name>
 		<no_trophies>Brak trofeów.
 Trofea można zdobywać, korzystając z aplikacji obsługującej funkcję trofeów.</no_trophies>
-		<not_earned>Nie zdobyto</not_earned>
-		<original>Oryginalne</original>
+		<not_earned>Niezdobyte</not_earned>
+		<original>Oryginalnie</original>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:842]: Revert both, is already good

File: lang/system/pl.xml [hunk 16]
```diff
@@ -358,24 +830,29 @@ System Vita3K jest gotowy!</completed_setup>
...
 		<sort>Sortuj</sort>
 		<trophies>Trofea</trophies>
-		<grade>stopień</grade>
+		<grade>Stopień</grade>
 		<progress>Postęp</progress>
-		<updated>Opdaterest</updated>
+		<updated>Zdobycie</updated>
```
Comment (Zangetsu38) [inline lang/system/pl.xml:847]:
```suggestion
		<updated>Zaktualizowano</updated>
```
Both is wrong here

File: lang/system/pl.xml (hunk 13 above)
Comment (mavethee) [reply lang/system/pl.xml:566]: Done

File: lang/system/pl.xml (hunk 12 above)
Comment (mavethee) [reply lang/system/pl.xml:560]: That's Sony's mistake but whatever already, not like I'm grammarly correct irl either.

File: lang/system/pl.xml (hunk 14 above)
Comment (mavethee) [reply lang/system/pl.xml:583]: Done, I was confused as Polish was 'Poland' other not really in that form.

File: lang/system/pl.xml [hunk 17]
```diff
@@ -384,10 +861,12 @@ Trofea można zdobywać, korzystając z aplikacji obsługującej funkcję trofe
 		<user_delete_warn>Użytkownik zostanie usunięty.
 Czy na pewno chcesz kontynuować?</user_delete_warn>
 		<user_deleted>Usunięto użytkownika.</user_deleted>
-		<choose_avatar>Wybierz awatara</choose_avatar>
+		<choose_avatar>Wybierz awatar</choose_avatar>
+		<reset_avatar>Resetuj awatar</reset_avatar>
 		<name>Nazwa</name>
-		<user>Uzytkownik</user>
+		<user>Użytkownik</user>
```
[the remark this answers is not in the document]
Comment (mavethee) [reply lang/system/pl.xml:867]: Please elaborate, I would understand that Vita has a typo, but PS4 has it definitely correct. It was missing special letter here.

[the remark this answers is not in the document]
Comment (Croden1999) [reply lang/system/pl.xml:867]: PS4 used  the "z" word, 'cause is edit user name.

File: lang/system/pl.xml [hunk 18]
```diff
@@ -358,24 +830,29 @@ System Vita3K jest gotowy!</completed_setup>
...
-		<not_earned>Nie zdobyto</not_earned>
+		<not_earned>Nie zdobyte</not_earned>
 		<original>Oryginalne</original>
 		<sort>Sortuj</sort>
 		<trophies>Trofea</trophies>
-		<grade>stopień</grade>
+		<grade>Stopień</grade>
 		<progress>Postęp</progress>
-		<updated>Opdaterest</updated>
+		<updated>Zaktualizowano</updated>
```
Comment (Croden1999) [inline lang/system/pl.xml:847]: Revert this line

File: lang/system/pl.xml [hunk 19]
```diff
@@ -314,7 +562,7 @@ System Vita3K jest gotowy!</completed_setup>
 		</date_time>
 		<language name="Język">
 			<system_language>Język systemu</system_language>
-			<input_language name="język wprowadzania">
+			<input_language name="Język wprowadzania">
```
Comment (Croden1999) [inline lang/system/pl.xml:565]: You're right.

```suggestion
			<input_language name="Języki wprowadzania">
```

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/system/pl.xml [hunk 20]
```diff
@@ -358,24 +830,29 @@ System Vita3K jest gotowy!</completed_setup>
...
+		<delete_trophy>Usuń trofeum</delete_trophy>
+		<trophy_deleted>Ta informacja o trofeum przypisana dla tego użytkownika zostanie usunięta.</trophy_deleted>
+		<locked>Niezdobyto</locked>
 		<details>Szczegóły</details>
 		<earned>Zdobyto</earned>
 		<name>Nazwa</name>
 		<no_trophies>Brak trofeów.
 Trofea można zdobywać, korzystając z aplikacji obsługującej funkcję trofeów.</no_trophies>
-		<not_earned>Nie zdobyto</not_earned>
+		<not_earned>Nie zdobyte</not_earned>
```
Comment (Croden1999) [inline lang/system/pl.xml:841]: Revert this line, here is "Nie zdobyto"

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/system/pl.xml (hunk 17 above)
[the remark this answers is not in the document]
Comment (mavethee) [reply lang/system/pl.xml:867]: > PS4 is used the "z" word, 'cause is edit user name, so no matter for this issue.

image
It's `Użytkownik` not `Uzytkownik`

File: lang/system/pl.xml (hunk 19 above)
Comment (mavethee) [reply lang/system/pl.xml:565]: Well was gathering Settings names from PS Vita User Guide site sooo

File: lang/system/pl.xml (hunk 17 above)
[the remark this answers is not in the document]
Comment (Croden1999) [reply lang/system/pl.xml:867]: not here, create new user you can see the name bar, is "Uzytkownik"

[the remark this answers is not in the document]
Comment (mavethee) [reply lang/system/pl.xml:867]: > not here, if you create new user and you can see the name bar, is "Uzytkownik"

@Croden1999 Ahh, it's default local usernames? Like User1, User2, User3?

[the remark this answers is not in the document]
Comment (mavethee) [reply lang/system/pl.xml:867]: Yeah its exactly that. Sorry for confusion. Changing back and should be ready for review.

File: lang/system/pl.xml (hunk 1 above)
Comment (mavethee) [reply lang/system/pl.xml]: Done.

**Comment by Croden1999** [comment]
lgtm

**Review by Croden1999 — APPROVED** [review]

**Comment by Zangetsu38** [comment]
> lgtm

not yet, i check some missing change request

File: lang/system/pl.xml (hunk 13 above)
Comment (Zangetsu38) [reply lang/system/pl.xml:565]: you have not done here
```suggestion
				<keyboards name="Klawiatury">
```

**Comment by mavethee** [comment]
> not yet, i check some missing change request

@Zangetsu38 should be fine now.

File: lang/system/pl.xml [hunk 21]
```diff
@@ -3,35 +3,83 @@
...
 			<install_license>Zainstaluj licencję</install_license>
 			<exit>Zamknij</exit>
 		</file>
-		<emulation name="Emulation">
-			<last_apps_used>Last Apps used</last_apps_used>
+		<emulation name="Emulacja">
+			<last_apps_used>Ostatnio używane aplikacje</last_apps_used>
 		</emulation>
-		<configuration name="Configuration">
+		<debug name="Debugowanie">
```
Comment (Croden1999) [inline lang/system/pl.xml:17]: here is not used the "name"

File: lang/system/pl.xml [hunk 22]
```diff
@@ -326,26 +573,250 @@ System Vita3K jest gotowy!</completed_setup>
...
+Może być użyteczne, jeśli posiadasz NIESTABILNE lub BARDZO WOLNE połączenie internetowe.</timeout_sleep_description>
+			<read_end_attempts>Próby zakończenia odczytu HTTP</read_end_attempts>
+			<read_end_attempts_description>Ilość prób do wykonania, gdy nie ma więcej danych do odczytu,
+niższa wartość może poprawić wydajność, ale może sprawić, że gry będą niestabilne, jeśli masz naprawdę słabe połączenie z internetem.</read_end_attempts_description>
+			<read_end_sleep>Uśpienie przy próbach zakończenia odczytu HTTP</read_end_sleep>
+			<read_end_sleep_description>Nakłada uśpienie, gdy nie ma więcej danych do odczytu,
+niższa wartość może poprawić wydajność, ale może sprawić, że gry będą niestabilne, jeśli masz naprawdę słabe połączenie z internetem.</read_end_sleep_description>
 		</network>
-		<close>Zamykać</close>
+		<debug name="Debugowanie">
```
Comment (Croden1999) [inline lang/system/pl.xml:793]: also here

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/system/pl.xml (hunk 13 above)
Comment (Croden1999) [reply lang/system/pl.xml]: @Zangetsu38 here is no problem here, is fine

File: lang/system/pl.xml (hunk 21 above)
Comment (mavethee) [reply lang/system/pl.xml:17]: @Croden1999 it's used indeed, shows up in menubar translated.

File: lang/system/pl.xml (hunk 13 above)
Comment (Zangetsu38) [reply lang/system/pl.xml]: Is already fixed

File: lang/system/pl.xml (hunk 22 above)
Comment (mavethee) [reply lang/system/pl.xml]: @Croden1999 This one doesn't have a name so I removed this one before, rebasing 13 commits did a thing or two. Will fix, but menubar thing should stay.

File: lang/system/pl.xml (hunk 21 above)
Comment (Croden1999) [reply lang/system/pl.xml:17]: but i already cancel this title in pre version

Comment (mavethee) [reply lang/system/pl.xml]: Okay whatever, this project will be polinglish anyways, could blindly copy english xml and not dealing with this out random, waiting for a merge.

Removed in the name of Winnie the Pooh.
image
W IMIĘ POLSKI LUDOWEJ!
Signing off, comrade! o7

**Review by Croden1999 — APPROVED** [review]

**Review by Zangetsu38 — APPROVED** [review]
Lgtm

**Review by Macdu — APPROVED** [review]

**Review by nishinji — APPROVED** [review]

**Pull request merged**